File: praktikum_algopascal/C/if.c
#include <stdio.h>

int main()
{
	int a, b;

	printf("Masukan Nilai A: ");
	scanf("%d", &a);

	printf("Masukan Nilai B: ");
	scanf("%d", &b);

	if (a == 5)
	{
		if (a < b)
		{
			printf("A anda lebih kecil dari B");
		}
		else
		{
			printf("A anda lebih besar dari B");
		}
	}
	else
		if (a > b)
		{
			printf("A anda lebih besar dari B");
		}
		else
		{
			printf("A anda lebih kecil dari B");
		}
	return 0;
}

File: praktikum_algopascal/pascal/if.pas
program if_ajjah;
	uses crt;
		var
			a: Integer;
			ket: string;
begin
	
	write('Masukan nilai A: ');
	readln(a);
	if a > 10 then
		write('Nilai lebih dari 10')
		else
		if a > 5 then
			write('Nilai lebih dari 5')
			else
				write('Nilai lebih kecil dari 5 atau 10');
end.